File: Mod13-Aplicacion/1.python/1.plantilla-aplicacion.py
# spark-submit --master local fichero.py rutaFichero

import sys
from pyspark import SparkContext
from pyspark import SparkConf

if __name__ == "__main__":
	# Si el faltan argumentos a ejecutar SALIR con ERROR
	if len(sys.argv) < 1:
		print >> sys.stderr, "ERROR: faltan argumentos"
		exit(-1)

	sconf = SparkConf()
	sconf.setAppName("Mi plantilla para programa")
	sconf.set("spark.ui.port","4141")

	sc = SparkContext(conf=sconf)

	"""
	aqui ira nuestro programa

	sc.textFile(sys.argv[1])

	"""

	sc.stop()	

File: Mod13-Aplicacion/2.scala/plantillaAplicacion/src/main/scala/stubs/1.plantilla-aplicacion.scala
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._
import org.apache.spark.SparkConf

// Cambiar mi programa por el nombre de tu aplicacion
object MiPrograma {
	
	def main(args: Array[String]){
		// Si el numero de argumentos no es correcto SALIR con ERROR
		if (args.length < 1){
			System.err.println("ERROR: faltan argumentos")
			System.exit(-1)
		}
	
		val sconf = new SparkConf()
		sconf.setAppName("mi plantilla programa")
		sconf.set("spark.ui.port","4141")
	
		val sc = new SparkContext(sconf)
/*
		Aqui va nuestro programa
*/
		// parar contexto
		sc.stop()
	}
}
